webapp/packages/plugin-navigation-tree/src/TreeNew/NodeControl.tsx

```diff
@@ -15,18 +15,18 @@ import { TreeDataContext } from './contexts/TreeDataContext.js';
 import type { NodeControlComponent } from './INodeRenderer.js';
 
 export const NodeControl: NodeControlComponent = observer(
-  forwardRef(function NodeControl({ nodeId }, ref) {
+  forwardRef(function NodeControl({ nodeId, className }, ref) {
     const data = useContext(TreeDataContext)!;
     const tree = useContext(TreeContext)!;
 
     const node = data.getNode(nodeId);
     const height = tree.getNodeHeight(nodeId);
 
     return (
-      <TreeNodeControl ref={ref} style={{ height }}>
+      <TreeNodeControl className={className} ref={ref} style={{ height }}>
```

Comment:
This is the inner tree component. It's not supposed to be used directly. It's meant to be easily re-implemented for customization.